website/src/components/views/compareSideBySide/GenericCompareSideBySidePage.astro

```diff
@@ -15,9 +15,10 @@ import { CompareSideBySideSelectorFallback } from '../../pageStateSelectors/Fall
 type OrganismViewCompareVariant = OrganismWithViewKey<typeof compareSideBySideViewKey>;
 interface Props {
     organism: OrganismViewCompareVariant;
+    unaligned?: boolean;
```

Comment:
Is this prop actually set anywhere?

Reply:
I use it later for Influenza - but not yet

Reply:
Should we remove it for now as it's currently still unused?